python/test_file_generator.py

```diff
@@ -1836,12 +1760,12 @@ def v673_5_file(self):
 		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
 
 	def v674_1_file(self):
-		"""Set origination charges to blank or -1."""
+		"""Set origination charges to '-1.'"""
 		name = "v674_1.txt"
 		path = self.validity_path
 		ts = self.ts_df.copy()
 		lar = self.lar_df.copy()
-		lar.origination_fee = lar.origination_fee.map(lambda x: random.choice(("-1", "")))
+		lar.origination_fee = -1
```

Comment:
Let's keep the data type as string here for consistency. Mixing types may change the representation in Pandas.